src/main/resources/assets/gtceu/lang/zh_cn.json

```diff
@@ -598,7 +598,7 @@
     "block.gtceu.lv_transformer_1a":"低压变压器（§7LV§r）",
     "block.gtceu.lv_transformer_2a":"低压进阶变压器（§7LV§r）",
     "block.gtceu.lv_transformer_4a":"低压高流变压器（§7LV§r）",
-    "block.gtceu.lv_wiremill":"基础线材轧机§r",
+    "block.gtceu.lv_wiremill":"基础线材轧机§",
```

Comment:
This change looks like it's probably a mistake. The original language file has the formatting reset in it.

Reply:
fixed